### PR TITLE
Clarify misleading statement regarding request for resources

### PR DESCRIPTION
**Pull request by zeitchef**
Clarifies a misleading statement that HTML makes a request for resources, rather than initiates an HTTP request to request resources.

### Description

Very minor change to clarify the entity responsible for requesting resources, and English grammar.

### Motivation

I'm excited to contribute to MDN and I'm motivated only in communicating as clearly as possible, especially for new developers.

### CODE REVIEW

File: files/en-us/web/performance/critical_rendering_path/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Optimizing the critical rendering path improves the time to first render. Unders
...
+A request for a web page or app starts with an HTTP request. The server returns the HTML, response headers and data. The browser then begins parsing the HTML, converting the received bytes to the DOM tree. The browser initiates requests every time it finds links to external resources, be it stylesheets, scripts, or embedded image references. Some requests are blocking, which means the parsing of the rest of the HTML is halted until the imported asset is handled. The browser continues to parse the HTML making requests and building the DOM, until it gets to the end, at which point it constructs the CSS object model. With the DOM and CSSOM complete, the browser builds the render tree, computing the styles for all the visible content. After the render tree is complete, layout occurs, defining the location and size of all the render tree elements. Once complete, the page is rendered, or 'painted' on the screen.
```
Comment (wbamberg) [inline files/en-us/web/performance/critical_rendering_path/index.md:20]:
```suggestion
A request for a web page or app starts with an HTTP request. The server sends a response containing the HTML. The browser then begins parsing the HTML, converting the received bytes to the DOM tree. The browser initiates requests every time it finds links to external resources, be it stylesheets, scripts, or embedded image references. Some requests are blocking, which means the parsing of the rest of the HTML is halted until the imported asset is handled. The browser continues to parse the HTML making requests and building the DOM, until it gets to the end, at which point it constructs the CSS object model. With the DOM and CSSOM complete, the browser builds the render tree, computing the styles for all the visible content. After the render tree is complete, layout occurs, defining the location and size of all the render tree elements. Once complete, the page is rendered, or 'painted' on the screen.
```

Thanks for your PR, @zeitchef !

The first part, HTML->HTTP, I like, but I don't think the second is right. It implies that the server returns three things: HTML, response headers, and data - but really of course the HTML is the data. I do think the current text is confusing, as well, so I suggested an alternative.

**Review by wbamberg — CHANGES_REQUESTED** [review]

File: files/en-us/web/performance/critical_rendering_path/index.md (hunk 1 above)
Comment (zeitchef) [reply files/en-us/web/performance/critical_rendering_path/index.md:20]: Thanks @wbamberg - makes perfect sense, and is much clearer. I'll commit the suggestion!

**Review by zeitchef — COMMENTED** [review]
👍🏻

**Review by wbamberg — APPROVED** [review]
👍 thank you @zeitchef !

**Pull request merged**